tests/ThisOutTest.php

```diff
@@ -85,6 +83,10 @@ public function getData(): array { return $this->data; }
                     '$data1===' => 'list<1>',
                     '$data2===' => 'list<2>',
                     '$data3===' => 'list<2|3>',
+                ],
+                'ignored_issues' => [
```

Comment:
as a general rule I like to not use `ignored_issues` for new issues, explicitly adding `@psalm-suppress` in the exact places where issues crop up